script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -65,6 +61,24 @@ function handleNetwork() {
     echo "[network: $NETWORK] diamond address found in deploy log file: $DIAMOND_ADDRESS"
   fi
 
+  # check if the diamond is already paused by calling owner() function and analyzing the response
+  local RESPONSE=$(cast call "$DIAMOND_ADDRESS" "owner()" --rpc-url "$RPC_URL")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)